app/Services/GoogleSheetsService.php

```
@@ -112,6 +114,12 @@ public function syncFormalization(string $spreadsheetId, string $sheetName, int
 
             try {
                 Formalization::updateOrCreate(['project_id' => $project->id], $record);
+
+                // Sempre sobrescreve (mesmo quando null): um NUP removido/corrigido
+                // na planilha precisa refletir em Opening, não ficar com valor antigo.
+                $nup = Import::normalizeNup($row[$config['opening_nup_column'] ?? 'N° DO PROCESSO (NUP)'] ?? null);
+                Opening::where('project_id', $project->id)->update(['opening_nup' => $nup]);
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\Opening::where().


(staticMethod.notFound)